public/api-merged.json

```diff
@@ -553,7 +553,7 @@
         "tags": [
           "ldap_debug"
         ],
-        "summary": "Returns the current state of the LDAP background sync integration",
```

Comment:
What are all the other changes for? They seem fine but not related to the PR

Reply:
These are currently being generated in `main` as well. This file is generated automatically but is not part of the build process. It's required to run it manually. It looks like it was not run for some changes that are already in `main`.

Reply:
This is cleaned up now after https://github.com/grafana/grafana/pull/55750 got merged and by regenerating swagger with the fix https://github.com/grafana/grafana-enterprise/pull/3895